public/app/core/logs_model.ts

```diff
@@ -186,7 +186,11 @@ export function makeSeriesForLogs(sortedRows: LogRowModel[], bucketSize: number,
 }
 
 function isLogsData(series: DataFrame) {
-  return series.fields.some(f => f.type === FieldType.time) && series.fields.some(f => f.type === FieldType.string);
+  return (
+    series.fields.length &&
```

Comment:
Also not sure that `series.fields.length` is necessary here. If `series.fields` is of length 0 then the following `series.fields.some(...)` expressions should evaluate to false

Reply:
Not needed, same as the comment above I've removed it.